src/website/webview/views.py

```diff
@@ -490,22 +490,57 @@ def get_context_data(self, **kwargs: Any) -> Any:
 
         # Major channels are the important channels that a user wants to keep an eye on.
         # FIXME make it dynamic
-        major_channels = ["nixos-23.11", "nixos-24.05", "nixos-24.11", "nixos-unstable"]
+        major_channels = ["23.11", "24.05", "24.11", "unstable"]
+
+        def get_major_channel(branch_name: str) -> str | None:
+            for mc in major_channels:
+                if mc in branch_name:
+                    return f"nixos-{mc}"
+            return None
 
         for obj in context["objects"]:
+            # TODO Validate assumption that there is only one container
```

Comment:
It's a wrong assumption, there are multiple containers, but we shall only select the most relevant one

Reply:
Any idea how to figure out the most relevant one?

Reply:
Hmm… I need to think, but basically you need to select the first container containing a package name or containing a CVSS vector string, this sort of stuff.

Maybe, the backend should do more preprocessing to make this palatable.

Reply:
correct!